### PR TITLE
docs: deprecated traceloop integration in favor of native otel

### PR DESCRIPTION
**Pull request by nirga**
## Title

Deprecated Traceloop integration in favor of the native OpenTelemetry integration for observability

## Type

📖 Documentation

## Changes

* Removed Traceloop integration code
* Traceloop integration docs -> native otel docs
* Fixed traceloop litellm proxy docs

## [REQUIRED] Testing - Attach a screenshot of any new tests passing locall
If UI changes, send a screenshot/GIF of working UI fixes

### CODE REVIEW

File: docs/my-website/docs/observability/opentelemetry_integration.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import Image from '@theme/IdealImage';
+
+# 🔭 OpenTelemetry - Tracing LLMs with any observability tool
```
Comment (ishaan-jaff) [inline docs/my-website/docs/observability/opentelemetry_integration.md]: can you remove the emoji ` 🔭 `

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
@nirga it looks like your linter made a bunch of changes to the docs making it hard to review changes made. Can you only make the changes to the doc you want (without linting)

File: docs/my-website/docs/proxy/logging.md [hunk 2]
```diff
@@ -479,57 +490,56 @@ This is the Span from OTEL Logging
...
+OTEL_ENDPOINT="https://api.traceloop.com"
+OTEL_HEADERS="Authorization=Bearer%20<your-api-key>"
 ```
 
-Add `otel` as a callback on your `litellm_config.yaml`
+**Step 2:** Add `otel` as a success_callback
 
 ```shell
 litellm_settings:
-  callbacks: ["otel"]
```
Comment (ishaan-jaff) [inline docs/my-website/docs/proxy/logging.md]: this should be `callbacks`

File: docs/my-website/docs/observability/opentelemetry_integration.md (hunk 1 above)
Comment (nirga) [reply docs/my-website/docs/observability/opentelemetry_integration.md]: @ishaan-jaff but everyone else has an emoji I want one too! 😅

Comment (ishaan-jaff) [reply docs/my-website/docs/observability/opentelemetry_integration.md]: Screenshot 2024-08-21 at 7 59 14 AM


We're removing emojis from all logging integrations - got feedback from litellm users that they don'y like emojis on docs

**Comment by nirga** [comment]
@ishaan-jaff fixed everything and removed the emoji!

File: docs/my-website/docs/proxy/logging.md (hunk 2 above)
Comment (ishaan-jaff) [reply docs/my-website/docs/proxy/logging.md]: 1. this should be callbacks @nirga 

2. Can you not replace the Honeycomb for Traceloop tab. We have users using our honeycomb integration.

File: docs/my-website/docs/observability/opentelemetry_integration.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+</TabItem>
+
+</Tabs>
+
+Use just 2 lines of code, to instantly log your LLM responses **across all providers** with OpenTelemetry:
+
+```python
+litellm.success_callback = ["otel"]
```
Comment (ishaan-jaff) [inline docs/my-website/docs/observability/opentelemetry_integration.md]: should be `callbacks`

File: docs/my-website/docs/observability/opentelemetry_integration.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Install the OpenTelemetry SDK:
+
+```
+pip install opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp
+```
+
+Set the environment variables (different providers may require different variables):
+
+<Tabs>
+
```
Comment (ishaan-jaff) [inline docs/my-website/docs/observability/opentelemetry_integration.md:23]: can you add the OTEL http, OTEL GRPC examples here too

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
requested changes

**Comment by nirga** [comment]
@ishaan-jaff done! 🙌

**Pull request merged**